datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+// [START create_fileset_entry]
+
+import com.google.cloud.datacatalog.ColumnSchema;
+import com.google.cloud.datacatalog.CreateEntryGroupRequest;
+import com.google.cloud.datacatalog.CreateEntryRequest;
+import com.google.cloud.datacatalog.Entry;
+import com.google.cloud.datacatalog.EntryGroup;
+import com.google.cloud.datacatalog.EntryGroupName;
+import com.google.cloud.datacatalog.EntryName;
+import com.google.cloud.datacatalog.EntryType;
+import com.google.cloud.datacatalog.GcsFilesetSpec;
+import com.google.cloud.datacatalog.LocationName;
+import com.google.cloud.datacatalog.Schema;
+
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+
+public class CreateFilesetEntry {
+
+  /**
+   * Create Fileset Entry
+   *
+   * @param projectId    The project ID to which the fileset belongs, e.g. 'my-project'
+   * @param entryGroupId The Entry Group ID to which the fileset belongs,
+   *                     e.g. 'fileset_entry_group'
+   * @param entryId      The Entry ID for the fileset, e.g. 'fileset_entry_id'
+   */
+  public static void createEntry(String projectId, String entryGroupId, String entryId) {
+
+    // -------------------------------
+    // Currently, Data Catalog stores metadata in the
+    // us-central1 region.
+    // -------------------------------
+    String location = "us-central1";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DataCatalogClient dataCatalogClient = DataCatalogClient.create()) {
+
+      // -------------------------------
+      // 1. Environment cleanup: delete pre-existing data.
+      // -------------------------------
+      // Delete any pre-existing Entry with the same name
+      // that will be used in step 3.
+      try {
+        dataCatalogClient.deleteEntry(
+                EntryName.of(projectId, location, entryGroupId, entryId).toString());
+      } catch (Exception e) {
```

Comment:
You shouldn't catch Exception - you should catch the specific error being thrown and show the user how to handle it correctly.

Reply:
Removed, will add new PRs for delete methods.